README.md

```diff
@@ -124,16 +124,16 @@ The `position` and `scale` are handled automatically by the system, but the anim
 The way to do this is by extending either the `Entity` or the `HealthManagedEntity` abstract classes in the `Hkmp.Game.Client.Entity` namespace.
 The `HealthManagedEntity` class extends the `Entity` with functionality to handle entities that make use of a `HealthManager` component internally to determine when to do.
 In most cases, the `HealthManagedEntity` will be the class to extend, but in some cases (for example for False Knight, see `FalseKnight.cs`) the dying sequences will be handled differently and thus not use the HealthManager component.
-Most if not all entities in Hollow Knight are controlled by finite state machines (`FSM`) implemented in C# in the library PlayMaker.
+Most if not all entities in Hollow Knight are controlled by finite state machines, (`FSM`) implemented in C# with the PlayMaker library.
 States in these FSMs have actions that are executed when the state is entered.
 Exact details are omitted here, but can probably be found in the modding channels in the [official HK discord](https://discord.gg/hollowknight) or the newer [HK modding discord](https://discord.gg/VDsg3HmWuB).
 To inspect the FSMs of entities, the tool [FSMViewAvalonia](https://github.com/nesrak1/FSMViewAvalonia) can be used.
   
 Both of the extendable classes contain abstract methods that need to be implemented, namely `InternalInitializeAsSceneHost`, `InternalInitializeAsSceneClient`, `InternalSwitchToSceneHost`, `UpdateAnimation`, and `UpdateState`.
 - `InternalInitializeAsSceneHost`: This method will be called when the entity is initialized given that the local player is a scene host.
   This means that the entity needs to send animation and state updates and let the underlying FSM run its course.  
-- `InternalInitializeAsSceneClient`: This method will be called when the entity is initialized given that the local player is a scene client.
-  This means that the entity needs to be disallowed to progress its FSM on its own, but rather await animation and state updates and act accordingly.
+- `InternalInitializeAsSceneClient`: This method will be called when the entity is initialized, given that the local player is a scene client.
+  This means that the entity must not be allowed to progress its FSM on its own, but rather await animation and state updates and act accordingly.
```

Comment:
While this might be better in terms of grammar, I feel that it doesn't convey the idea as much that we need to make sure that the FSM explicitly does not progress anymore (since its normal state is to progress).

Reply:
FSM mean finite state machine right? In Unity we mean states of animation or player states. Could we say, "We need to make sure the entity can not progress to another FSM state on it's own, but rather wait for animation and state updates to change it's state."